mdsuite/visualizer/d2_data_visualization.py

```diff
@@ -40,7 +40,7 @@ class DataVisualizer2D:
     Visualizer for two-dimensional data.
     """
 
-    def __init__(self, title: str):
+    def __init__(self, title: str, path: str):
```

Comment:
Can you add a doc string for the path.

Reply:
Can you check the type hints first - as far as I know https://github.com/zincware/MDSuite/blob/293a96e549dbd142c879c18034b1db40b8932171/mdsuite/experiment/experiment.py#L231-L233 is it a pathlib.Path